jax/lib/__init__.py

```diff
@@ -43,7 +43,7 @@ def _check_jaxlib_version():
 
 _check_jaxlib_version()
 
-
+from jaxlib import tpu_client
```

Comment:
I think we need to commit the BUILD.bazel + install_xla_in_source_tree.sh first, build a new jaxlib, and then commit the rest of the changes?

Reply:
Thanks for the suggestion @skye, I have updated the PR to push the BUILD.bazel + install_xla_in_source_tree.sh.